### DISCUSSION TITLE
Question over Broker Console Access and local development

### DISCUSSION THREAD

**Discussion by gw-kamara** [arkmq-org/arkmq-org-broker-operator#787] [Q&A]
As i understand, The console is only available from within the same name namespace

I can see in the jolokia-access.xml that this is set already 

`    <cors>
        
        <allow-origin>*://artemis-broker-ss-0.artemis-broker-hdls-svc.default.svc.cluster.local*</allow-origin>
`
        
 Im not concerned about the url its disposable. 
 
 Using the artemis_single.yml file example I've added 
 
 `  console:
    expose: true
    `
    
  But I am still not able to get access to the interface by running a 
  `k port-forward ...`
  as the connection is refused. 
  
  How am i able to modify this to allow access from anywhere (I currently have no security concerns as it's development)
  
  Thanks in advance

**Comment by brusdev** [comment]
@gw-kamara for dev purposes, you could add an entry in your /etc/hosts file and use the ingress created by the operator i.e. to access http://artemis-broker-wconsj-0-svc-ing.apps.artemiscloud.io/ you need to add the following entry in your /etc/hosts:
```
<KUBERNETES_IP> artemis-broker-wconsj-0-svc-ing.apps.artemiscloud.io
```

**Reply by gw-kamara** [reply]
I got around this by using traefik and creating host route to the "artemis-broker-hdls-svc" service and seems to work very well. 

Different problem is once the brokers are scaled beyond 1 - I unable to log in anymore to the console (logs report success but the page just refreshes) ...

**Reply by brusdev** [reply]
@gw-kamara why are you not using the ingress created by the operator?

**Reply by Syndlex** [reply]
I think they did not use the normal ingress because Traefik doesn't allow it to have tls-passthrough from a normal Ingress. It needs a IngressRouteTCP as far as i know.
